.github/workflows/azure-static-web-apps-salmon-pond-055c7fa10.yml

```diff
@@ -18,6 +18,10 @@ jobs:
       - uses: actions/checkout@v2
         with:
           submodules: true
+      - run: npm install
+      - run: npx playwright install
+      - run: npm start & npx wait-on http://localhost:3000

```

Comment:
Sorry to jump in! I think it would be cool to use the actual environment where it was deployed at. Then it's ensured that the deployment configuration is as expected and the dev vs. prod server are usually quite different.

See here how to obtain the URL after the environment was deployed: https://github.com/Azure/static-web-apps-deploy/blob/v1/action.yml#L43

Basically output parameter of the deploy action and afterwards you can run the tests.